### PR TITLE
Fix greeting

### LINKED ISSUE #612: fix: greeting in /debug page
image

Right now in the it doesn't show the greeting even if the user is logged in

### PR DESCRIPTION
**Pull request by shubhdevelop**
### Issue

Closes #612 

### What is the change?

Provide a small description in points of what did you change
fixes navbar, navbar now shows greeting when user is logged in

### Is Development Tested?

- [ ] Yes
- [x] No

### Before / After Change Screenshots
Before :
image

After :
image

### CODE REVIEW

File: debug.html [hunk 1]
```diff
@@ -280,9 +280,16 @@ <h4>Apply privileges :</h4>
       </p>
     </footer>
     <script src="/js/utils.js"></script>
+    <script src="js/dropdown.js"></script>
     <script type="module" src="/js/user.js"></script>
+    <script src="https://api.mapbox.com/mapbox-gl-js/v2.10.0/mapbox-gl.js"></script>
+    <script type="module" src="js/debug.js"></script>
+    <script type="module" src="js/animate.js"></script>
```
Comment (shubhamsigdar1) [inline debug.html:287]: Does this is required in debug page??

File: debug.html [hunk 2]
```diff
@@ -280,9 +280,16 @@ <h4>Apply privileges :</h4>
...
     <script type="module" src="./js/navbar.js"></script>
-    <script type="module" src="./js/debug.js"></script>
+    <script src="js/scrollBtn.js"></script>
```
Comment (shubhamsigdar1) [inline debug.html:289]: Does this is required in debug page??

File: debug.html [hunk 3]
```diff
@@ -280,9 +280,16 @@ <h4>Apply privileges :</h4>
...
+    <script src="js/darkmode.js"></script>
+    <script src="js/featureFlag.js"></script>
+    <script type="module" src="js/map.js"></script>
```
Comment (shubhamsigdar1) [inline debug.html:292]: Does this is required in debug page??

File: debug.html [hunk 4]
```diff
@@ -280,9 +280,16 @@ <h4>Apply privileges :</h4>
...
     <script src="/js/utils.js"></script>
+    <script src="js/dropdown.js"></script>
     <script type="module" src="/js/user.js"></script>
+    <script src="https://api.mapbox.com/mapbox-gl-js/v2.10.0/mapbox-gl.js"></script>
+    <script type="module" src="js/debug.js"></script>
+    <script type="module" src="js/animate.js"></script>
     <script type="module" src="./js/navbar.js"></script>
-    <script type="module" src="./js/debug.js"></script>
+    <script src="js/scrollBtn.js"></script>
+    <script src="js/darkmode.js"></script>
```
Comment (shubhamsigdar1) [inline debug.html:290]: Does this is required in debug page??

File: debug.html [hunk 5]
```diff
@@ -280,9 +280,16 @@ <h4>Apply privileges :</h4>
       </p>
     </footer>
     <script src="/js/utils.js"></script>
+    <script src="js/dropdown.js"></script>
     <script type="module" src="/js/user.js"></script>
+    <script src="https://api.mapbox.com/mapbox-gl-js/v2.10.0/mapbox-gl.js"></script>
```
Comment (shubhamsigdar1) [inline debug.html:285]: Does this is required in debug page??

**Review by shubhamsigdar1 — COMMENTED** [review]
There is Linting error
image

File: debug.html (hunk 1 above)
Comment (shubhdevelop) [reply debug.html:287]: mapbox and animate.js no

File: debug.html (hunk 2 above)
Comment (shubhdevelop) [reply debug.html:289]: scroll.btn.js no

File: debug.html (hunk 3 above)
Comment (shubhdevelop) [reply debug.html:292]: no

**Comment by shubhdevelop** [comment]
> There is Linting error 

linting is failing due to module.export in `utlils.js`

image

even in the index.html page, this in console giving  the error that `module is not defined`

This export is for testing setup in my opinion as this file is not used anywhere as module

**Comment by ravikumar1002** [comment]
> > There is Linting error
> 
> linting is failing due to module.export in `utlils.js`
> 
> image
> 
> even in the index.html page, this in console giving the error that `module is not defined`
> 
> This export is for testing setup in my opinion as this file is not used anywhere as module

yes, but due to index.test.js file I have fixed that issue in my PR which I will raise today

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**